tests/integration_tests/functional/test_mmds.py

```diff
@@ -825,7 +825,7 @@ def test_mmds_snapshot(bin_cloner_path,  version):
     # artifact.
     firecracker_artifacts = artifacts.firecrackers(
         # v1.0.0 breaks snapshot compatibility with older versions.
```

Comment:
Nit: outdated comment

Reply:
ooo good catch. I also removed some code that specifically referred to 1.0.0